### PR TITLE
Allow specifying $aadToken audience

### PR DESCRIPTION
**Pull request by flanakin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -369,14 +369,16 @@ Content-Type: application/json
...
+* `{{$aadToken [new] [public|cn|de|us|ppe] [<domain|tenantId>] [aud:<domain|tenantId>]}}`: Add an Azure Active Directory token based on the following options (must be specified in order):
 
   `new`: Optional. Specify `new` to force re-authentication and get a new token for the specified directory. Default: Reuse previous token for the specified directory from an in-memory cache. Expired tokens are refreshed automatically. (Use `F1 > Rest Client: Clear Azure AD Token Cache` or restart Visual Studio Code to clear the cache.)
 
   `public|cn|de|us|ppe`: Optional. Specify top-level domain (TLD) to get a token for the specified government cloud, `public` for the public cloud, or `ppe` for internal testing. Default: TLD of the REST endpoint; `public` if not valid.
 
   `<domain|tenantId>`: Optional. Domain or tenant id for the directory to sign in to. Default: Pick a directory from a dropdown or press `Esc` to use the home directory (`common` for Microsoft Account).
 
+  `aud:<domain|tenantId>`: Optional. Target Azure AD app id (aka client id) or domain the token should be created for (aka audience or resource). Default: Domain of the REST endpoint.
```
Comment (Huachao) [inline README.md]: Could please help to add the case in your `aadToken.http`?

**Comment by flanakin** [comment]
Not yet. Been busy. Should have an update next week.

**Comment by flanakin** [comment]
Finally got back to this. Sorry about that. Lots of stuff going on :-P The good news is, this helped me find a bug :-)

**Comment by Huachao** [comment]
@flanakin Merged, thanks. Glad to hear that, and thanks for your contribution for enable AAD in my extension.

**Pull request merged**